_script/build

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+cd $(dirname $0)/..
+
+function go_version_check {
+	go_version=$(go version  | cut -d " " -f 3)
+	supported_go_version="go1.9"
```

Comment:
shall we be a bit more flexible and let higher versions (future) of GO? I mean comparing with something like '>=' or having a list of supported_go_versions (array)

Reply:
Maybe... but this checks only if you are generating the binary with the expected go version. And we will want to have full control of which version is being used to publish a new binary/docker/ospackage, so this flexibility probably does not apply here. That's just my view ;)